packages/p2p/src/pages/app.jsx

```diff
@@ -31,10 +31,15 @@ const App = () => {
 
     const lang = getLanguage();
 
+    const [is_app_loading, setIsAppLoading] = React.useState(true);
     const [order_id, setOrderId] = React.useState(null);
     const [action_param, setActionParam] = React.useState();
     const [code_param, setCodeParam] = React.useState();
 
+    React.useEffect(() => {
+        if (!is_loading) setIsAppLoading(false);
+    }, [is_loading]);
+
```

Comment:
Here we are adding `isAppLoading` state in order to prevent blinking (because we are firing onMount (which is setting `is_loading` into `true`) in the useEffect, we are rendering p2p content for a few ms), that's why we are adding `isAppLoading` state.

Reply:
@wojciech-deriv seems like it works, thanks for pointing it